integration-tests/src/tests/client/features/stateless_validation.rs

```diff
@@ -383,11 +385,7 @@ fn test_chunk_state_witness_bad_shard_id() {
     // Client should reject this ChunkStateWitness and the error message should mention "shard"
     tracing::info!(target: "test", "Processing invalid ChunkStateWitness");
     let signer = env.clients[0].validator_signer.get();
-    let res = env.clients[0].process_chunk_state_witness(witness, witness_size, None, signer);
-    let error = res.unwrap_err();
-    let error_message = format!("{}", error).to_lowercase();
-    tracing::info!(target: "test", "error message: {}", error_message);
-    assert!(error_message.contains("shard"));
+    let _res = env.clients[0].process_chunk_state_witness(witness, witness_size, None, signer);
```

Comment:
There's a panic from `get_shard_index()` after switching to V2.

Reply:
Ah that's pretty bad. Feel free to either:
1) Fix it (may be complicated / lots of code if you need to add error handling)
2) Leave as is but put a TODO(wacban) in there instead of FIXME and I will have a look. 
3) Make the default shard layout V1 (hopefully this works?)

Reply:
I'll try 3 (should probably work from the look of the code) which seems like a nice middle ground before finishing transition to V2